source/brailleDisplayDrivers/eurobraille.py

```diff
@@ -132,8 +137,137 @@
 	0x0f:"Esytime 32 standard",
 	0x10:"Esytime evo 32",
 	0x11:"Esytime evo 32 standard",
+	0x12: "b.note",
+	0x13: "b.note 2",
+	0x14: "b.book",
+	0x15: "b.book 2"
 }
 
+defaultGestureMapEntries = {

```

Comment:
considering you are moving this dictionary, would you consider moving it to a separate file and structuring the driver in a similar way to the albatross driver folder?

This requires:
- an `__init__` file that exports the `BrailleDisplayDriver`
- moving the bulk of the driver file to a `driver.py` file
- moving the gestures to a `gestures.py` file?

Reply:
I looked at the structure of the albatross display driver and applied it to the eurobraille driver